pyproject.toml

```diff
@@ -17,6 +17,7 @@ dependencies=[
     "networkx==3.1",
     "PyYAML",
     "tree-sitter",
+    "gqlalchemy",
```

Comment:
any expectations on a min. version?

Reply:
I'm not certain, I've tested with 1.4.0 and 1.5.0 and know they both work with our current system. To prevent breaking changes, we can pin it to 1.5.0 which is the most recent version.

Reply:
If we know 1.4.0 and 1.5.0 both work, let's just use 1.4.0 as the min. working version.

Reply:
It looks like there is actually a dependency version conflict with networkx. 

```
skema[all] 0.1.0 depends on networkx==3.1
gqlalchemy 1.4.0 depends on networkx<3.0.0 and >=2.5.1
```

Do you know if the version can be lowered?

Reply:
Ah, sounds like the min. compatible version might actually be `1.5.0`.  If we put `>=` for the `gqlalchemy` version (ex. `"gqlachemy>=1.5.0"`), does everything install for your locally?

Reply:
The same thing happens with 1.5.0 unfortunately. My skema install is from before we started pinning dependencies, so I have an earlier version of networkx.